Source/CarthageKit/FrameworkExtensions.swift

```diff
@@ -273,7 +274,9 @@ extension URL {
 
 extension FileManager: ReactiveExtensionsProvider {
 	@available(*, deprecated, message: "Use reactive.enumerator instead")
-	public func carthage_enumerator(at url: URL, includingPropertiesForKeys keys: [URLResourceKey]? = nil, options: FileManager.DirectoryEnumerationOptions = [], catchErrors: Bool = false) -> SignalProducer<(FileManager.DirectoryEnumerator, URL), CarthageError> {
+	public func carthage_enumerator(at url: URL, includingPropertiesForKeys keys: [URLResourceKey]? = nil,
+	                                options: FileManager.DirectoryEnumerationOptions = [],
+	                                catchErrors: Bool = false) -> SignalProducer<(FileManager.DirectoryEnumerator, URL), CarthageError> {
```

Comment:
Thanks, we agreed on the style from the Swift Standard Library like this:

```swift
public func carthage_enumerator(
		at url: URL, includingPropertiesForKeys keys: [URLResourceKey]? = nil,
		options: FileManager.DirectoryEnumerationOptions = [],
		catchErrors: Bool = false
	) -> SignalProducer<(FileManager.DirectoryEnumerator, URL), CarthageError> {
		return reactive.enumerator(at: url, includingPropertiesForKeys: keys, options: options, catchErrors: catchErrors)
	}
```

I've updated this, too.